.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why was this added?

Reply:
Without this, the package.json file gave an error in vscode. The error was:
"["Problems loading reference 'vscode://schemas/settings/configurationDefaults': Unable to load schema from 'vscode://schemas/settings/configurationDefaults': cannot open vscode://schemas/settings/configurationDefaults. Detail: Unable to resolve text model content for resource vscode://schemas/settings/configurationDefaults."]"

Reply:
also added another line in same file in latest commit as it was showing error for settings.json upon restarting the vscode.

Reply:
I don't think you should include this in the pr. It sounds like an issue with your local setup. Please remove this file.